explorer/templates/explorer/fullscreen.html

```diff
@@ -1,26 +1,25 @@
 {% load i18n static %}
+{% load vite %}
```

Comment:
What I've mentioned about changes to the static files impacts on this template tag, but makes things a bit more like regular django.

Although maybe I don't know enough about vite, and maybe I'm just not doing things properly when I'm testing this. I've mostly been running the vite build, then copying the files into the static directory and then the app into a project like it would if it'd come from pypi.